src/pymor/models/iosys.py

```diff
@@ -122,6 +122,9 @@ def mag_plot(self, w, mu=None, ax=None, ord=None, Hz=False, dB=False, **mpl_kwar
 
         return out
 
+    def _solve_for_solution(self, mu=None):
```

Comment:
Is this defined in the model interface?

Reply:
This was a leftover from the last PR. I have removed it.